### PR TITLE
switch controller manager to generated clients

### PR DESCRIPTION
**Pull request by deads2k**
Switches the controller manager to generated clients.

@ncdc ptal

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33103)

### CODE REVIEW

**Comment by deads2k** [comment]
@k8s-bot unit test this: issue #32770

File: cmd/kube-controller-manager/app/controllermanager.go [hunk 1]
```diff
@@ -159,11 +158,11 @@ func Run(s *options.CMServer) error {
 
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartLogging(glog.Infof)
-	eventBroadcaster.StartRecordingToSink(kubeClient.Events(""))
+	eventBroadcaster.StartRecordingToSink(&unversionedcore.EventSinkImpl{Interface: kubeClient.Core().Events("")})
```
Comment (ncdc) [inline cmd/kube-controller-manager/app/controllermanager.go]: Why the change to EventSinkImpl?

**Review by ncdc — CHANGES_REQUESTED** [review]

File: cmd/kube-controller-manager/app/controllermanager.go (hunk 1 above)
Comment (deads2k) [inline cmd/kube-controller-manager/app/controllermanager.go]: > Why the change to EventSinkImpl?

This uses the generated clientset, the old impl used the manual client.

Comment (ncdc) [inline cmd/kube-controller-manager/app/controllermanager.go]: Argh, `kubeClient.Core().Events("")` almost conforms to `pkg/client/record.EventSink` except for `Patch` :-(

**Review by ncdc — APPROVED** [review]

File: cmd/kube-controller-manager/app/controllermanager.go (hunk 1 above)
Comment (deads2k) [inline cmd/kube-controller-manager/app/controllermanager.go]: > Argh, kubeClient.Core().Events("") almost conforms to pkg/client/record.EventSink except for Patch :-(

Yeah, I tried to fix that once before, but it was nasty.  I think I'll have better luck going down this route and perhaps ripping out the event recorder entirely.
